components/organisation-unit-tree/src/organisation-unit-node/use-org-data/use-org-data.js

```diff
@@ -35,35 +35,25 @@ export const useOrgData = (
 
     const defaultData = { id, displayName }
     const variables = { id, isUserDataViewFallback }
-    const [state, setState] = useState({
-        loading: true,
-        error: null,
-        data: defaultData,
+    const { loading, error, data } = useDataQuery(ORG_DATA_QUERY, {
```

Comment:
Using the useDataQuery's values directly instead of with the intermediate state.